src/common/util/src/file_util.cpp

```diff
@@ -32,6 +33,9 @@
 #    endif
 /// @brief Windows-specific 'mkdir' wrapper
 #    define makedir(dir) _mkdir(dir)
+#    ifdef OPENVINO_ENABLE_UNICODE_PATH_SUPPORT
+#        define wmakedir(dir) _wmkdir(dir)
+#    endif
```

Comment:
```suggestion
#    ifdef OPENVINO_ENABLE_UNICODE_PATH_SUPPORT
#        define makedir(dir) _wmkdir(dir)
#    else
#        define makedir(dir) _mkdir(dir)
#    endif
```

It can reduce `#ifdef` usage later in code as `makedis` will use string or w string depends if unicode path supported.